<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>764</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="asime">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>751</width>
      <height>531</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Eliminación Gaussiana</string>
     </attribute>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>0</y>
        <width>751</width>
        <height>91</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>30</y>
         <width>221</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Eliminación Gausiana</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
     <widget class="QTextEdit" name="gausiana1">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>130</y>
        <width>331</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="gaussiana2">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>190</y>
        <width>331</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>140</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Valores de A</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>200</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Valores de B</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>320</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Resultado :  </string>
      </property>
     </widget>
     <widget class="QPushButton" name="agaus">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>390</y>
        <width>201</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Aceptar</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>330</y>
        <width>55</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Potencia Inversa</string>
     </attribute>
     <widget class="QFrame" name="frame_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>100</y>
        <width>741</width>
        <height>401</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>50</y>
         <width>41</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Matriz </string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>100</y>
         <width>141</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Numero de iteraciones</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>160</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Tolerancia</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>210</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Resultado :</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="potencia1">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>50</y>
         <width>331</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QTextEdit" name="potencia2">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>100</y>
         <width>331</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QTextEdit" name="potencia3">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>150</y>
         <width>331</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="ainve">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>280</y>
         <width>161</width>
         <height>71</height>
        </rect>
       </property>
       <property name="text">
        <string>Procesar</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>40</y>
        <width>221</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Potencia Inversa</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Potencia Simetrica</string>
     </attribute>
     <widget class="QFrame" name="frame_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>120</y>
        <width>741</width>
        <height>361</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>60</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Matriz simetrica</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>110</y>
         <width>141</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Numero de iteraciones</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>160</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Tolerancia</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>220</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Resultado :</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="ps1">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>50</y>
         <width>331</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QTextEdit" name="ps2">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>100</y>
         <width>331</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QTextEdit" name="ps3">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>150</y>
         <width>331</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="asimetrica">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>290</y>
         <width>161</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string>Procesar</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>50</y>
        <width>221</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Potencia Simetrica</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Interpolación de Lagrange</string>
     </attribute>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>40</y>
        <width>221</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Interpolación de Lagrange</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
